src/cs50/__init__.py

```diff
@@ -1,5 +1,18 @@
 import os
 import sys
+import warnings
+
+from .cs50 import eprint, get_char, get_float, get_int, get_string
+try:
+    from .cs50 import get_long
+except:
```

Comment:
`except ImportError`

bare `except`s catch things like `KeyboardInterrupt` and `SystemExit`

Reply:
Thanks for this, btw!